resources/assets/js/components/CreateProject.js

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import {browserHistory} from 'react-router';
+import MyGlobleSetting from './MyGlobleSetting';
+
+
+class CreateProject extends Component {
+  constructor(props){
+    super(props);
+    this.state = {name: '', information: '', deadline: '', type: '', status: ''};
+    this.handleChange1 = this.handleChange1.bind(this);
+    this.handleChange2 = this.handleChange2.bind(this);
+    this.handleChange3 = this.handleChange3.bind(this);
+    this.handleChange4 = this.handleChange4.bind(this);
+    this.handleChange5 = this.handleChange5.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange1(e){
```

Comment:
I think it would be better to give this event handler a better name. It seems to be used for changes in the name field, so maybe `handleChangeProjectName` would be good.

The same applies for the other event handler functions here.